### PR TITLE
First pass of mongo schema

### PR DESCRIPTION
**Pull request by ives1227**
**Initial Mongo schema**
* * *

**JIRA Ticket**: [(link)](https://github.com/harvard-lts/HDC/issues/22)

* Other Relevant Links (Mailing list discussion, related pull requests, etc.)
MVP Document: https://docs.google.com/document/d/1VYkGjiT7cdLtDi8lzsdpuIXVl8N-eU3BNGzxMNmkPb8/edit#heading=h.o4vjapab0ipm

# What does this Pull Request do?
Adds a first pass at a Mongo Schema for the import management to use.

# How should this be tested?

No testing necessary.

# Test coverage
Yes/No: Are changes in this pull-request covered by:
- unit tests? No
- integration tests? No

### CODE REVIEW

File: mongo/schema.json [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+      ],
+      "description": "The name of the application making the deposit."
+    },
+    "ingest_provenance" : {
+      "type": "object",
+      "properties": {
+        "ingest_status": {
+          "description": "The status of the object being ingested",
+          "enum": [
+            "",
```
Comment (GPortas) [inline mongo/schema.json:22]: Is this empty string enum value correct?

**Review by GPortas — CHANGES_REQUESTED** [review]
Looks good!

Just a small comment related to ingest_status enum.

File: mongo/schema.json (hunk 1 above)
Comment (ives1227) [reply mongo/schema.json]: @GPortas that is a good catch - it has no purpose in this workflow.  I've removed it.

**Comment by ives1227** [comment]
@GPortas I changed the import status enum.  This should be ready.  Thanks!

**Review by GPortas — APPROVED** [review]
Looks great! 👍  Thank you @ives1227

**Pull request merged**